README.md

```diff
@@ -44,12 +44,10 @@ Check the exmaple
 
 ### Using it as a library
  - Add it as a library with
-`git submodule add https://github.com/LuisMayo/objection_engine.git`
+`https://github.com/LuisMayo/objection_engine.git`
```

Comment:
The depends on which project format the consumer uses, not sure the best way to describe that.

Reply:
Maybe we could give a couple of examples?

> If using pip
> `git+https://github.com/LuisMayo/objection_engine.git`
> If using X
> `EXAMPLE`